src/Logic/__test__/operate.test.js

```diff
@@ -0,0 +1,16 @@
+import operate from '../operate';
+
+describe('operations', () => {
+  test('it adds numbers', () => {
+    expect(operate(2, 2, '+')).toBe('4');
+  });
+  test('it subtracts numbers', () => {
+    expect(operate(2, 2, '-')).toBe('0');
+  });
+  test('it multiplies numbers', () => {
+    expect(operate(2, 2, 'x')).toBe('4');
+  });
+  test('it divides numbers', () => {
+    expect(operate(2, 2, '÷')).toBe('1');
+  });
```

Comment:
-[ ] great job with the operations.. kindly add an operation test for modulo`(%)`

Reply:
[tufoinnkuo10](https://github.com/tufoinnkuo10), thanks for your review. I'll make the changes.